src/server.js

```diff
@@ -149,10 +149,22 @@ function init(socket, client) {
 
 		socket.on("disconnect", function() {
 			client.clientDetach(socket.id);
+			sendConnectionInfo(client);
 		});
 		client.clientAttach(socket.id);
 
 		socket.on(
+			"remote-sign-out",
+			function(data) {
+				var connection_list = Object.keys(client.attachedClients);
+				for (var i = 0; i < connection_list.length; i++) {
+					if (data.socket_id === connection_list[i]) {
+						manager.sockets.of("/").connected[connection_list[i]].emit("sign-out", {});
```

Comment:
Idea was, emitting event to client, client disconnects itself and then server gets disconnection event. This way you get confirmation that client got disconnected. Just emitting disconnection event wont log out the client and you see recent history there.